### PR TITLE
Make agent hostname configurable

### PR DESCRIPTION
**Pull request by zzambers**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ To run the whole thing, simply run make:
 ```
 make
 ```
+to set agent hostname, required by some tests, use AGENT_HOSTNAME parameter. (Agent expects to have same krb setup as expected by [jck](https://web.archive.org/web/20201126185131/https://icedtea.classpath.org/wiki/JCKDistilled#kerberos_prep). Use your domain for service prinicpal, e.g. http/service.example.com).
+```
+make AGENT_HOSTNAME=agent.example.com
```
Comment (judovana) [inline README.md:19]: Waht will happen when the variable is not set?

Comment (zzambers) [reply README.md:19]: It will fail somewhere unless `SKIP_AGENT_TESTS=1`.  :)

Comment (judovana) [reply README.md:19]: The exception then will be very criptic (if any)
Just thoughts - include our include our principla setup script. Provide some default aka "you .have.not.set.your.agent.useagent.hostname.var.or.skip.agent.tests" hmm?

Comment (zzambers) [reply README.md:19]: I am thinking about creating check in beginning of krb tests, which would fail them, but give some instructions..

Comment (judovana) [reply README.md:19]: that would be enough. ty. its ok as additional chnageset.

Comment (zzambers) [reply README.md:19]: Addressed in another PR: https://github.com/rh-openjdk/CryptoTest/pull/50

**Pull request merged**